packages/nuxt/src/imports/directives.ts

```diff
@@ -0,0 +1,63 @@
+import { createUnplugin } from 'unplugin'
+import type { AddonsOptions, Import } from 'unimport'
+import { createUnimport } from 'unimport'
+import type { ImportPresetWithDeprecation } from 'nuxt/schema'
+import MagicString from 'magic-string'
+import { isVue } from '../core/utils'
+
+export const DirectivesPlugin = ({
+  addons,
+  dirs,
+  imports,
+  presets,
+}: {
+  addons: AddonsOptions
+  dirs: string[]
+  imports: Import[]
+  presets: ImportPresetWithDeprecation[]
+}) => createUnplugin(() => {
+  const useImports: Import[] = []
+  function visit (i: Import) {
+    if (i.meta?.vueDirective === true) {
+      useImports.push(i)
+    }
+  }
+  imports?.forEach(visit)
+  presets?.forEach((preset) => {
```

Comment:
review this, I guess not necessary (getting some errors starting nuxt dev server, re-checkthis without `?`)